loopj/jquery-tokeninput
# bug fixed: Pressing the [Enter] key causes AJAX results not to show.

Test case:
1. Configure tokenInput to get results remotely.
2. Start typing and press [Enter] before the results come back -> you end up with "No Results" showing in the dropdown.
3. Delete your search text.
4. Retype it.
5. Now you can see your results as they are already cached and show right away.

Added a "pendingSearch" flag to ignore any [Enter] key presses while a remote search is in progress.
